src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Request as a Service",
+    "version": "4.0",
+    "title": "Concur Request API Documentation"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/travelrequest/v4",
+  "tags": [
+    {
+      "name": "Request Resource",
+      "description": "Manage documents used for pre-spend authorizations within Concur Request"
```

Comment:
Manage Request which is representing the starting point of pre-spend authorizations within Concur Request